babel.config.js

```diff
@@ -13,38 +14,26 @@ const useESModules = ['esm'].includes(process.env.BABEL_ENV)
 
 const presets = [
   [
-    '@babel/preset-env',
+    '@babel/preset-react',
     {
-      // bugfixes: true,
-      // browserslistEnv: process.env.BABEL_ENV || process.env.NODE_ENV,
-      modules: useESModules ? false : 'commonjs',
-      // shippedProposals: api.env('modern'),
+      runtime: 'automatic',
     },
   ],
+  '@babel/preset-typescript',
   [
-    '@babel/preset-react',
+    '@babel/preset-env',
     {
-      runtime: 'automatic',
+      modules: useESModules ? false : 'commonjs',
+      targets: {
+        browsers: 'defaults, not dead, not IE 11, not op_mini all',
+      },
     },
   ],
 ]
 
 const plugins = [
   'babel-plugin-optimize-clsx',
-  // Need the following 3 proposals for all targets in .browserslistrc.
-  // With our usage the transpiled loose mode is equivalent to spec mode.
-  ['@babel/plugin-proposal-class-properties', { loose: true }],
-  ['@babel/plugin-proposal-private-methods', { loose: true }],
-  ['@babel/plugin-proposal-private-property-in-object', { loose: true }],
-  ['@babel/plugin-proposal-object-rest-spread', { loose: true }],
```

Comment:
Are these not needed @alexanderflink? They seem to still live in package.json :)

Reply:
I removed them when updating the babel config since I figured they aren't needed for the browser targets we have. Not entirely sure how to test this though. I'll also remove them from package.json!

Reply:
Awesome! Makes sense to remove them from package.json if they are not used in babel.config. Make sure to remove them though since I can still find them under `resolutions` in package.json ;) @alexanderflink